app/controllers/group_members_controller.rb

```diff
@@ -90,10 +90,10 @@ def create
   # DELETE /group_members/1
   # DELETE /group_members/1.json
   def destroy
-
+    noticeMessage = "Group member was successfully removed."
     @group_member.destroy
     respond_to do |format|
-      format.html { redirect_to group_path(@group_member.group), notice: 'Group member was successfully destroyed.' }
+      format.html { redirect_to group_path(@group_member.group), notice: noticeMessage }
```

Comment:
- Do not use camelCase for variables in ruby, use snake_case
- No need of temporary variable here, just inline the message

Reply:
Codeclimate gave an error of line length being too long. That's why I had to introduce a variable.

Reply:
Ill update accordingly.